### ISSUE TITLE
CI builds failing on NPM_TOKEN

### ISSUE THREAD

**Issue by rosskevin** [ctavan/jupp#24]
I'd PR but it looks like something that needs privileges on travis to change:

```
Error: Failed to replace env in config: ${NPM_TOKEN}
    at /home/travis/.nvm/versions/node/v8.9.1/lib/node_modules/npm/lib/config/core.js:418:13
    at String.replace (<anonymous>)
```

**Comment by rosskevin** [comment]
@ctavan I'm also happy to come on as a contributor if you want the help.

**Comment by rosskevin** [comment]
Closed by #25
